packages/atomic/src/components/atomic-smart-snippet/atomic-smart-snippet-suggestions.tsx

```diff
@@ -0,0 +1,258 @@
+import {Component, h, Prop, State, Element} from '@stencil/core';
+import {
+  InitializableComponent,
+  InitializeBindings,
+  Bindings,
+  BindStateToController,
+} from '../../utils/initialization-utils';
+import ArrowRight from '../../images/arrow-right.svg';
+import ArrowDown from '../../images/arrow-down.svg';
+import {
+  buildInteractiveResult,
+  buildSmartSnippetQuestionsList,
+  SmartSnippetQuestionsList,
+  SmartSnippetQuestionsListState,
+  SmartSnippetRelatedQuestion,
+} from '@coveo/headless';
+import {Hidden} from '../common/hidden';
+import {Heading} from '../common/heading';
+import {LinkWithResultAnalytics} from '../result-link/result-link';
+import {Button} from '../common/button';
+import {randomID} from '../../utils/utils';
+
+/**
+ * The `atomic-smart-snippet-suggestions-suggestions` component displays an accordion of questions related to the query with their corresponding answers.
+ *
+ * You can style the snippets by inserting a template element like this:
+ * ```html
+ * <atomic-smart-snippet-suggestions>
+ *   <template>
+ *     <style>
+ *       b {
+ *         color: blue;
+ *       }
+ *     </style>
+ *   </template>
+ * </atomic-smart-snippet-suggestions>
+ * ```
+ *
+ * @part container - The wrapper with a border around the entire component.
+ * @part heading - The heading above the list of related question.
+ * @part questions - The list of related questions.
+ * @part question-answer-expanded - An expanded related question.
+ * @part question-answer-collapsed - A collapsed related question.
+ * @part question-expanded-button - The button to collapse a related question.
+ * @part question-collapsed-button - The button to expand a related question.
+ * @part question-expanded-icon - The caret of an expanded related question.
+ * @part question-collapsed-icon - The caret of a collapsed related question.
+ * @part question-expanded-text - The title of an expanded related question.
+ * @part question-collapsed-text - The title of a collapsed related question.
```

Comment:
I added a `-expanded` and `-collapsed` version of most parts that are related to a relatedQuestion because I don't see any other way that those could be styled if a client wants a special expanded or collapsed style. We could also be more opinionated here and simplify the parts. What do you think?

Reply:
how about splitting the parts, so instead of `question-collapsed-text` you could have `question-text collapsed`. this way you could target either both at the same time or them separately, and it's easier to read. Could possibly have less descriptions either and explain about this feature in one place? something like
```
@part question-button - The button to collapse a related question. Also appends 'expanded' or 'collapsed' to its part for more granular control.
```

I wasn't sure if this suggestion would work, so I made a snippet
![Screenshot 2022-05-03 at 10 39 44](https://user-images.githubusercontent.com/30511433/166432845-660046f9-9947-4635-984d-67a9f437f838.png)

Reply:
oh right, that makes more sense! I'll change it.